dff/pipeline/pipeline/component.py

```diff
@@ -70,21 +70,30 @@ def __init__(
         name: Optional[str] = None,
         path: Optional[str] = None,
     ):
-        #: Maximum component execution time (in seconds),
-        #: if it exceeds this time, it is interrupted (for asynchronous only!).
         self.timeout = timeout
-        #: Requested asynchronous property; if not defined, :py:attr:`~requested_async_flag` is used instead.
+        """
+        Maximum component execution time (in seconds),
+        if it exceeds this time, it is interrupted (for asynchronous only!).
+        """
         self.requested_async_flag = requested_async_flag
-        #: Calculated asynchronous property, whether the component can be asynchronous or not.
+        """Requested asynchronous property; if not defined, :py:attr:`~requested_async_flag` is used instead."""
         self.calculated_async_flag = calculated_async_flag
-        #: Component start condition that is invoked before each component execution;
-        #: component is executed only if it returns `True`.
+        """Calculated asynchronous property, whether the component can be asynchronous or not."""
```

Comment:
AFAIK, in english we say "if [something] or not" or "whether [something]" (without "not").

Reply:
If I understand you, it is still okay to write that way: https://mcelroytutoring.com/blog-post.php?id=4165

Reply:
Both "whether [something]" and "whether [something] or not" are acceptable:
https://www.merriam-webster.com/dictionary/whether
https://dictionary.cambridge.org/dictionary/english/whether